src/index.js

```diff
@@ -1,19 +1,22 @@
-import React from 'react'
+import React, { Suspense } from 'react'
 import ReactDOM from 'react-dom'
-import Pages from './pages'
 import * as serviceWorkerRegistration from './serviceWorkerRegistration'
 import reportWebVitals from './reportWebVitals'
 import { Theme } from './styles'
 import { ApolloProvider } from '@apollo/client'
 import { client } from './services/GraphQl'
 import ErrorBoundary from './layouts/ErrorBoundary'
+import LoadingView from './components/LoadingView'
+const Pages = React.lazy(() => import('./pages'))
 
 ReactDOM.render(
   <React.StrictMode>
     <Theme>
       <ApolloProvider client={client}>
         <ErrorBoundary>
-          <Pages />
+          <Suspense fallback={<LoadingView />}>
+            <Pages />
+          </Suspense>
```

Comment:
I have noticed too that still "Loading..." message appears before loading view (that you have develop in this pull request), I prefer to remove this message or replace this message for the new page that you have to develop.

I would like to know what do you think: is better "Suspense" than the actual logic in "Loading..." message? Do they do the same thing? What is better in this case?

@Eghinner @Israel-Laguan @e-azocar

Reply:
I dont have experience working with `React Suspence` but I have readed about this, and is a experimental feature, it isn't full, however, looks like a good idea use it

Reply:
Well, I'm still not sure how to proceed. I understand that I'm going ahead with React Suspence. 
What should I correct?
My component shows a "Loading..." text? It's weird I didn't add any text

Reply:
Ok! Let's use `suspense`. @Eghinner We previously created the logic to display "Loading..." while loading the page. Can you remove this logic so that only your change was displayed?

Reply:
@Katsu08 I took an issue where I can solve the "Loading..." issue. I would like to do that part separately